### PR TITLE
Show right messages as soon as possible

### PR DESCRIPTION
**Pull request by x4base**
1. Flashed messages should be flushed in every single page, otherwise they will be accumulated and shown in the wrong page, making the users extremely confused.
2. The error messages of API calls should be shown immediately, and should not flash any message, because it will always be shown in the wrong page.

### CODE REVIEW

File: caravel/assets/javascripts/global_functions.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+var $ = window.$ = require('jquery');
```
Comment (mistercrunch) [inline caravel/assets/javascripts/global_functions.js]: I think this belongs in caravel.js or utils.js

File: caravel/views.py [hunk 2]
```diff
@@ -1100,13 +1137,7 @@ def refresh_datasources(self):
...
     @app.errorhandler(500)
     def show_traceback(self):
-        if config.get("SHOW_STACKTRACE"):
-            error_msg = traceback.format_exc()
-        else:
-            error_msg = "FATAL ERROR\n"
-            error_msg = (
-                "Stacktrace is hidden. Change the SHOW_STACKTRACE "
-                "configuration setting to enable it")
+        error_msg = get_error_msg()
```
Comment (mistercrunch) [inline caravel/views.py]: Why removing the stacktrace?

File: caravel/assets/javascripts/global_functions.js (hunk 1 above)
Comment (x4base) [inline caravel/assets/javascripts/global_functions.js]: I can move showApiMessage() and toggleCheckbox() into utils.js. 
But we still need to include some .entry.js in list.html, right? Should I still include global_functions.entry.js (or maybe rename it to list.entry.js) and call util.toggleChekbox() in this file? Or do you have any other idea?

Comment (x4base) [inline caravel/assets/javascripts/global_functions.js]: Or are you only talking about the line you mentioned? I only have to replace
```
var $ = window.$ = require('jquery');
var jQuery = window.jQuery = $;
```
with 
```
var $ = require('jquery');
```
, just don't touch the window variable, right?

**Comment by x4base** [comment]
I've just found out it shouldn't be in list.html, otherwise the script will be included multiple times if there are multiple inline list views. For example, in the datasource edit page, there are column list tab and metric list tab, so the script will be included twice. I think it should be in base.html, so I named it common.entry.js

**Pull request merged**